Makefile

```diff
@@ -397,7 +409,7 @@ flushvarnish: guard-DEPLOY_TARGET
 
 # This internal target has been created to have the good global variable values
 # from rc_XXX file.
-flushvarnishinternal: guard-VARNISH_HOSTS guard-API_URL guard-E2E_TARGETURL
+flushvarnishinternal: guard-API_URL guard-E2E_TARGETURL
```

Comment:
why remove `guard-VARNISH_HOSTS`?

Reply:
rc_ dev hasn't VARNISH_HOSTS

Reply:
You can use an empty list for dev

Reply:
ok

Reply:
I have still keep the guard-VARNISH_HOSTS but I have mprove the log message when VARNISH_HOST is empty

Reply:
because "export VARNISH_HOSTS=" in rc_dev was detected as a failure

Reply:
I was thinking `export VARNISH_HOSTS=()` but you changed that and it would be `export VARNISH_HOSTS=""`

LGTM

Reply:
it didn't work with VARNISH_HOSTS="", I've alaso tried with a () but didn't work too. You can try if you want.